coralproject/cay
# Remove react warnings

## What does this PR do?

I updated React the other day and it spawned a bunch of warnings. I upgraded some packages and some code so the console isn't as much of a mess during development
## How do I test this PR?
- `npm install`
- move around the Ask pages and see if there are warnings
- there are some vestigial warnings about `react-router`, but I'll deal with it later.

@coralproject/frontend
